modules/contact/src/main/kotlin/edu/stanford/spezi/modules/contact/ContactComposable.kt

```diff
@@ -76,14 +76,13 @@ fun ContactComposable(contact: Contact, modifier: Modifier = Modifier) {
                 horizontalArrangement = Arrangement.spacedBy(Spacings.small),
                 verticalAlignment = Alignment.CenterVertically
             ) {
-                val image = contact.image ?: Icons.Default.AccountBox
```

Comment:
just a small detail, but maybe a minor imporevement. currently the image in the `data class contact` is nullable and has the default value `null`, wouldn't it be worth considering removing the nullability and  setting the default value of the `contact` directly to `ImageResource.Vector(Icons.Default.AccountBox)` and then save us the null check at this point to set a Default Value or is it planned to use the image in other ways where it is needed to be "really" null? @pauljohanneskraft

Reply:
I like the idea of providing a default value and getting rid of the nullability 🙌

Reply:
@pauljohanneskraft  fyi: I have decided to remove the nullabllity 👍